app/views/404.hbs

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Streetmix - Page not Found</title>
```

Comment:
Let's rename this title to "Not found - Streetmix".

1. Usually, it's better to have the more specific title name first, rather than the more generic site name. This makes the tab easier to identify in a list of tabs.
2. Generally, we will use [sentence case](https://streetmix.readme.io/docs/content-style-guide#section-sentence-case) rather than mixed capitalization or Title Case.